specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -423,8 +469,118 @@ def verify_kzg_proof_multi_impl(commitment: KZGCommitment,
     ]))
 ```
 
+#### `verify_cell_kzg_proof_batch_impl`
+
+```python
+def verify_cell_kzg_proof_batch_impl(row_commitments: Sequence[KZGCommitment],
+                                     row_indices: Sequence[RowIndex],
+                                     column_indices: Sequence[ColumnIndex],
+                                     cosets_evals: Sequence[CosetEvals],
+                                     proofs: Sequence[KZGProof]) -> bool:
+    """
+    Verify a set of cells, given their corresponding proofs and their coordinates (row_index, column_index) in the blob
+    matrix. The i-th cell is in row row_indices[i] and in column column_indices[i].
+    The list of all commitments is provided in row_commitments_bytes.
+
+    This function implements the universal verification equation introduced here:
+    https://ethresear.ch/t/a-universal-verification-equation-for-data-availability-sampling/13240
```

Comment:
Perhaps let's link to the relevant func instead of the blog post here. The main func should link to the blog post.

Reply:
What do you mean by relevant func in this context?

Sure, I will add the link to the blog post to `verify_cell_kzg_proof_batch`.